hw/ip/kmac/data/kmac_base_testplan.hjson

```diff
@@ -265,7 +265,17 @@
     {
       name: error_cg
       desc: '''
-            Covers that all errors have been seen by the KMAC.
+            Covers various error scenarios:
```

Comment:
Why doesn't it cover all errors? Perhaps add the reason.

Reply:
I believe this is just a phrasing difference on my end, all errors are still being covered - I will revert the wording to make it more explicit, thanks!